realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -176,3 +176,18 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeIsAttached
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_LinkView_nativeFind
+  (JNIEnv *env, jobject, jlong nativeLinkViewPtr, jlong rowIndex)
+{
+    TR_ENTER_PTR(nativeLinkViewPtr)
+    try {
+        LinkView *lv = LV(nativeLinkViewPtr);
+        if (!ROW_INDEX_VALID(env, lv, rowIndex)) {
```

Comment:
This checking is not correct as well. It should crash just like table_view before. Should be `ROW_INDEX_VALID(env, &lv->get_target_table(), rowIndex)` . Unless i missed something...